Tests/AppStoreConnectClientTests/TestHelpers/MockObjects.swift

```diff
@@ -26,13 +27,82 @@ import OpenAPIRuntime
 @testable import AppStoreConnectClient
 
 enum MockObjects {
-    static var errorResponse: Components.Schemas.ErrorResponse {
-        let response = Components.Schemas.ErrorResponse(
-            errors: Components.Schemas.ErrorResponse.errorsPayload()
+    static var errorBadRequest: Components.Schemas.ErrorResponse {
+        return (
+            .init(
+                errors: Components.Schemas.ErrorResponse.errorsPayload.init(
+                    arrayLiteral: .init(
+                        status: "400",
+                        code: "BAD_REQUEST",
+                        title: "Invalid Input",
+                        detail: "The provided data failed validation"
+                    )
+                )
+            )
+        )
+    }
+    
+    static var errorForbidden: Components.Schemas.ErrorResponse {
+        return (
+            .init(
+                errors: Components.Schemas.ErrorResponse.errorsPayload.init(
+                    arrayLiteral: .init(
+                        status: "403",
+                        code: "FORBIDDEN",
+                        title: "Access Denied",
+                        detail: "You do not have permission to access this resource"
+                    )
+                )
+            )
+        )
+    }
+    
+    static var errorNotFound: Components.Schemas.ErrorResponse {
+        return (
+            .init(
+                errors: Components.Schemas.ErrorResponse.errorsPayload.init(
+                    arrayLiteral: .init(
+                        status: "404",
+                        code: "NOT_FOUND",
+                        title: "Resource Not Found",
+                        detail: "The requested resource was not found"
+                    )
+                )
+            )
         )
-        return response
     }
     
+    static var errorUnauthorized: Components.Schemas.ErrorResponse {
+        return (
+            .init(
+                errors: Components.Schemas.ErrorResponse.errorsPayload.init(
+                    arrayLiteral: .init(
+                        status: "401",
+                        code: "UNAUTHORIZED",
+                        title: "Unauthorized",
+                        detail: "Authentication credentials were missing or incorrect"
+                    )
+                )
+            )
+        )
+    }
+    
+    static var errorResponseArray: Components.Schemas.ErrorResponse = {
+        let payload1 = Components.Schemas.ErrorResponse.errorsPayloadPayload(
+            status: "400",
+            code: "BED_REQUEST",
+            title: "Server Error",
+            detail: "An unexpected error"
+        )
+        let payload2 = Components.Schemas.ErrorResponse.errorsPayloadPayload(
+            status: "500",
+            code: "INTERNAL_SERVER_ERROR",
```

Comment:
Need some comment regarding the source of these codes (INTERNAL_SERVER_ERROR)

Reply:
Changed.